apps/sms/views/inbox/test/unit/startup_test.js

```diff
@@ -0,0 +1,90 @@
+/*global InboxView,
+         InterInstanceEventDispatcher,
+         LazyLoader,
+         MessageManager,
+         MocksHelper,
+         Navigation,
+         Settings,
+         Startup,
+         TimeHeaders
+*/
+
+'use strict';
+
+require('/views/shared/test/unit/mock_time_headers.js');
+require('/views/shared/test/unit/mock_navigation.js');
+require('/views/shared/test/unit/mock_settings.js');
+require('/views/shared/test/unit/mock_inter_instance_event_dispatcher.js');
+require('/views/shared/test/unit/mock_inbox.js');
+require('/shared/test/unit/mocks/mock_lazy_loader.js');
+require('/services/test/unit/mock_message_manager.js');
+
+require('/views/inbox/js/startup.js');
+
+var MocksHelperForInboxStartup = new MocksHelper([
+  'InboxView',
+  'InterInstanceEventDispatcher',
+  'LazyLoader',
+  'MessageManager',
+  'Navigation',
+  'Settings',
+  'TimeHeaders'
+]).init();
+
+suite('InboxView Startup', function() {
+  MocksHelperForInboxStartup.attachTestHelpers();
+
+  setup(function() {
+    this.sinon.spy(MessageManager, 'init');
+    this.sinon.spy(Navigation, 'init');
+    this.sinon.spy(InboxView, 'init');
+    this.sinon.spy(TimeHeaders, 'init');
+    this.sinon.spy(Settings, 'init');
+    this.sinon.spy(Navigation, 'setReady');
+    this.sinon.spy(Navigation, 'toDefaultPanel');
+    this.sinon.spy(InterInstanceEventDispatcher, 'connect');
+    this.sinon.stub(InboxView, 'once');
+    this.sinon.stub(LazyLoader, 'load').returns(Promise.resolve());
+
+    var gaiaHeader = document.createElement('gaia-header');
+    gaiaHeader.setAttribute('no-font-fit', '');
+    document.body.appendChild(gaiaHeader);
+
+    Startup.init();
+  });
+
+  test('correctly initializes dependencies', function() {
+    sinon.assert.calledOnce(MessageManager.init);
```

Comment:
Maybe these checks are too verbose and not required, tell me if you don't want simpler tests that test only something important.